apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -103,7 +108,7 @@ testBothBookers.describe("pro user", () => {
     await page.goto("/bookings/upcoming");
     await page.locator('[data-testid="cancel"]').first().click();
     await page.waitForURL((url) => {
-      return url.pathname.startsWith("/booking");
+      return url.pathname.startsWith("/booking/");
```

Comment:
`/booking` also matches `/bookings/upcoming` but we wanted to match /booking/something